29th_first_semina/app/src/main/java/com/example/a29th_first_semina/ui/view/follwer/FollowerAdapter.kt

```diff
@@ -0,0 +1,41 @@
+package com.example.a29th_first_semina.ui.view.follwer
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.a29th_first_semina.databinding.ItemFollowerListBinding
+
+class FollowerAdapter : RecyclerView.Adapter<FollowerAdapter.FollowerViewHolder>() {
+
+    val follwerList = mutableListOf<FollowerData>()
+
+    class FollowerViewHolder(private val binding: ItemFollowerListBinding) :
+        RecyclerView.ViewHolder(binding.root) {
+        fun onBind(data: FollowerData) {
+            with(binding){
+                ivProfile.setImageResource(data.image)
+                tvName.text = data.name
+                tvIntro.text = data.intro
+            }
+        }
+    }
+
+    override fun onCreateViewHolder(
+        parent: ViewGroup,
+        viewType: Int
+    ): FollowerViewHolder {
+        val binding = ItemFollowerListBinding.inflate(
+            LayoutInflater.from(parent.context),
+            parent, false
+        )
+        return FollowerViewHolder(binding)
+
+    }
+
+    override fun onBindViewHolder(holder: FollowerViewHolder, position: Int) {
+        holder.onBind(follwerList[position])
+
+    }
+
+    override fun getItemCount(): Int = follwerList.size
```

Comment:
```suggestion
    override fun getItemCount() = follwerList.size
```

코틀린에는 타입 자동추론기능이 있으니 적극 활용해보시는걸 추천드립니다